build/zlib-1.2.8/contrib/vstudio/vc14/zlibstat.vcxproj

```diff
@@ -212,26 +185,27 @@
       <WarningLevel>Level3</WarningLevel>
       <SuppressStartupBanner>true</SuppressStartupBanner>
       <DebugInformationFormat>OldStyle</DebugInformationFormat>
-      <RuntimeLibrary>MultiThreadedDebugDLL</RuntimeLibrary>
+      <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
```

Comment:
Actually... I'm not sure it is fully correct:
https://msdn.microsoft.com/en-us/library/2kzt1wy3.aspx

It seems that "multithreadedDLL" is the preferred way (and the default value), and "multithreaded" is just to create an executable without dependencies.
The thing is that the different parts of the application (libs and dlls) should use the same runtime mode.
As such, I will revert that change.